package.json

```diff
@@ -2,26 +2,28 @@
   "name": "loopback-getting-started",
   "version": "1.0.0",
   "main": "server/server.js",
+  "engines": {
+    "node": ">=6"
+  },
   "scripts": {
     "lint": "eslint .",
     "start": "node .",
-    "posttest": "npm run lint && nsp check"
+    "posttest": "npm run lint"
```

Comment:
hmm any reason to remove nsp?

Reply:
nsp has shutdown: https://blog.npmjs.org/post/175511531085/the-node-security-platform-service-is-shutting. so we shouldn't be using it.